docs/settingUp.md

```diff
@@ -64,6 +81,10 @@ Eclipse IDE is our preferred development environment. Support requests related t
    * Text encoding: Go to `Eclipse → Preferences → General → Workspace`, change the `Text file encoding` setting from `Default` to `Other: UTF-8`.
    * JRE: Go to `Eclipse → Preferences → Java → Installed JRE` and ensure a JDK (not a JRE) is selected (use **JDK 1.7**, as recommended by GAE).
      One of the items in the [troubleshooting guide](troubleshooting-guide.md) explains how to do this.
+     
+   **Verification** : After this the make sure when you go to `Window`→`Preferences`→`Java`→`Installed Jre` and click on the JDK and choose edit which looks something like this :
```

Comment:
The idea is there, but the choice of words are quite weird. Consider the following alternative:

Go to `Eclipse → Preferences → Java → Installed JREs`, select the JDK 1.7 entry, and choose `Edit`. It should result in something like this: